components/script/dom/cssstyledeclaration.rs

```diff
@@ -10,7 +10,7 @@ use dom::bindings::js::{JS, Root};
 use dom::bindings::utils::{Reflector, reflect_dom_object};
 use dom::document::DocumentHelpers;
 use dom::element::{ElementHelpers, StylePriority};
-use dom::htmlelement::HTMLElement;
+use dom::element::Element;
```

Comment:
I'm afraid we introduced a new lint about sorting `use` statements that decided to bite you; sorry about that. Merging those two `use` lines should make it happy.